en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
+  * [Airframes](airframes/README.md)
+    * [Airframes Reference](airframes/airframe_reference.md)
+    * [Adding a New Airframe](airframes/adding_a_new_frame.md)
+    * [Parrot Bebop](advanced/parrot_bebop.md)
+  * [Porting Guide](debug/porting-guide.md)
+  * [Data Links](data_links/README.md)
+    * [SiK Radio](data_links/sik_radio.md)
```

Comment:
@bkueng  Propose to move [SikRadio](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/data_links/sik_radio.html) to user guide - not developer content.

Reply:
Ok but the Wifi doc is very similar and should be moved too then: https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/tutorials/telemetry.html. The pages can probably be merged.